wire-grpc-client/src/main/java/com/squareup/wire/GrpcStreamingCall.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire
+
+import kotlinx.coroutines.channels.ReceiveChannel
+import kotlinx.coroutines.channels.SendChannel
+import okio.Timeout
+import java.io.IOException
+
+interface GrpcStreamingCall<S : Any, R : Any> {
+  val timeout: Timeout
+  fun cancel()
+  fun execute(): Pair<SendChannel<S>, ReceiveChannel<R>>
+  fun executeBlocking(): Pair<MessageSink<S>, MessageSource<R>>
+  fun enqueue(callback: Callback<S, R>)
```

Comment:
Do we need enqueue for streaming calls? Won't it just always return immediately?

Reply:
Yeah, we could return channels that explode on use